docs/_src/dos-and-donts.css

```diff
@@ -0,0 +1,106 @@
+/* --- Wrapper for Side-by-Side Layout --- */
+.dos-and-donts {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1rem;
+  margin: 1em 0;
+}
+
+/* Mobile Responsiveness */
+@media screen and (max-width: 768px) {
+  .dos-and-donts {
+    flex-direction: column;
+  }
+}
+
+/* --- Common Card Styling (.dos and .donts) --- */
+.dos, .donts {
+  flex: 1;
+  min-width: 0;
+  background-color: var(--md-default-bg-color);
+  border: 1px solid var(--md-table-border-color);
+  border-top-width: 4px;
+  border-radius: 6px;
+
+  /* REQUESTED CHANGE: No horizontal padding, small vertical padding */
+  padding: 0.5rem 0;
+
+  box-shadow: 0 2px 4px rgba(0,0,0,0.05);
+  margin-bottom: 1em;
+}
+
+/* Remove bottom margin when inside the wrapper */
+.dos-and-donts .dos,
+.dos-and-donts .donts {
+  margin-bottom: 0;
+}
+
+/* --- Accessible Hidden Title --- */
+.dos h4, .donts h4 {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+/* --- List Styling --- */
+.dos ul, .donts ul {
+  list-style: none;
+
+  /* REQUESTED CHANGE: No margin on UL */
+  margin: 0 !important;
+}
+
+.dos li, .donts li {
+  display: flex;
+  align-items: flex-start;
+  gap: 8px;
+  margin-bottom: 0.4rem;
+  line-height: 1.4;
+  color: var(--md-footer-fg-color);
+}
+
+.dos li:last-child, .donts li:last-child {
+  margin-bottom: 0;
+}
+
+/* --- Icons (Shared Base) --- */
+.dos li::before, .donts li::before {
+  content: "";
+  display: inline-block;
+  flex-shrink: 0;
+  width: 18px;
+  height: 18px;
+  margin-top: 2px;
+  -webkit-mask-repeat: no-repeat;
+  mask-repeat: no-repeat;
+  -webkit-mask-position: center;
+  mask-position: center;
+}
+
+/* --- "DO" Specifics --- */
+.dos {
+  border-top-color: var(--md-admonition-success-color);
+}
+
+.dos li::before {
+  background-color: var(--md-admonition-success-color);
+  -webkit-mask-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' fill='none' stroke='black' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'%3E%3Ccircle cx='12' cy='12' r='10'/%3E%3Cpath d='M9 12l2 2 4-4'/%3E%3C/svg%3E");
```

Comment:
@Killusions Could you add the proper element icons for the dos and don'ts instead of the generated SVG 🙏

Reply:
@kfenner Done, though we have to copy the svg code as our way of including icons in the docs theme is not via css or class, so this is the only way.

Also improved the styling a bit.